### PR TITLE
fix(testing): add support for playwright --last-failed

### LINKED ISSUE #28096: Playwright --last-failed CLI flag not supported (1.44+)
### Current Behavior

Looking at https://github.com/nrwl/nx/blob/master/packages/playwright/src/executors/playwright/playwright.impl.ts#L10, seems like it's missing in the options + args list.

### Expected Behavior

One should be able to run playwright with `--last-failed` flag

### GitHub Repo

_No response_

### Steps to Reproduce

1. Have an NX project with playwright plugin
2. Try to configure `lastFailed` 
3. See that it isn't supported

### Nx Report

```shell
n/a
```

### Failure Logs

_No response_

### Package Manager Version

_No response_

### Operating System

- [X] macOS
- [ ] Linux
- [ ] Windows
- [ ] Other (Please specify)

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by agoldis**
`--last-failed` allows running only the failed tests from the previous run.
Example: `playwright test --last-failed`. [Documentation](https://playwright.dev/docs/test-cli#reference)

## Current Behavior

Cannot provide `--last-failed` flag to playwright

## Expected Behavior

Should be able provide `--last-failed` via configuration entry `lastFailed: boolean`

## Related Issue(s)

Fixes #28096

### CODE REVIEW

File: packages/playwright/src/executors/playwright/schema.json [hunk 1]
```diff
@@ -21,6 +21,10 @@
       "type": "boolean",
       "description": "Run tests with Playwright Inspector. Shortcut for 'PWDEBUG=1' environment variable and '--timeout=0',--max-failures=1 --headed --workers=1' options"
     },
+    "failedOnly": {
```
Comment (leosvelperez) [inline packages/playwright/src/executors/playwright/schema.json:24]:
```suggestion
    "lastFailed": {
```

File: packages/playwright/src/executors/playwright/playwright.impl.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ export interface PlaywrightExecutorSchema {
   browser?: 'all' | 'chromium' | 'firefox' | 'webkit' | string;
   config?: string;
   debug?: boolean;
+  failedOnly?: boolean;
```
Comment (leosvelperez) [inline packages/playwright/src/executors/playwright/playwright.impl.ts:18]:
```suggestion
  lastFailed?: boolean;
```

File: docs/generated/packages/playwright/executors/playwright.json [hunk 3]
```diff
@@ -24,6 +24,10 @@
         "type": "boolean",
         "description": "Run tests with Playwright Inspector. Shortcut for 'PWDEBUG=1' environment variable and '--timeout=0',--max-failures=1 --headed --workers=1' options"
       },
+      "failedOnly": {
```
Comment (leosvelperez) [inline docs/generated/packages/playwright/executors/playwright.json:27]:
```suggestion
      "lastFailed": {
```

**Review by leosvelperez — CHANGES_REQUESTED** [review]
Thanks for contributing!

There's a mismatch between what the PR describes to do and the change made. The correct option name accepted by Playwright (and mentioned in the PR title and description) is `--last-failed`, but the changes are adding a `failedOnly` option.

**Comment by agoldis** [comment]
@leosvelperez 🤦 my bad...

**Comment by agoldis** [comment]
@leosvelperez thanks for the quick review and the suggestions - it's all fixed now!

**Review by leosvelperez — APPROVED** [review]
Once again, thanks for the contribution!

**Pull request merged**